main.go

```diff
@@ -251,6 +257,14 @@ func (s Step) ProcessConfig() (Config, error) {
 	log.Printf("* device_destination: %s", deviceDestination)
 	fmt.Println()
 
+	if input.TestRepetitionMode != none && xcodeMajorVersion < 13 {
+		return Config{}, errors.New("test_repetition_mode is not supported below Xcode 13")
```

Comment:
Please include the title of the input also (like: `Export UITest Artifacts (export_uitest_artifacts)`).